src/elements/Chip/ChipTheme.ts

```diff
@@ -117,53 +117,53 @@ export const lightChipTheme: ChipTheme = {
       focus: 'focus-visible:outline-gray-500'
     },
     primary: {
-      base: 'bg-primary-700 border-primary-700 text-primary-700',
+      base: 'bg-primary border-primary text-primary',
       selectable: [
         baseTheme.colors.primary.selectable,
-        'hover:bg-primary-600 hover:border-primary-600'
+        'hover:bg-primary-hover hover:border-primary-hover'
       ].join(' '),
-      selected: '!bg-primary-600 text-white',
-      focus: 'focus-visible:outline-primary-500'
+      selected: '!bg-primary text-white',
```

Comment:
We have selected state for Chip, but idk what token we should use for this.
currently "active" token has same color with default.